src/Text/Pandoc/Writers/Powerpoint/Output.hs

```diff
@@ -1411,19 +1411,24 @@ getDefaultTableStyle = do
   return $ findAttr (QName "def" Nothing Nothing) tblStyleLst
 
 graphicToElement :: PandocMonad m => Integer -> Graphic -> P m Element
-graphicToElement tableWidth (Tbl tblPr hdrCells rows) = do
-  let colWidths = if null hdrCells
-                  then case rows of
+graphicToElement tableWidth (Tbl widths tblPr hdrCells rows) = do
+  let totalWidth = sum widths
+  P.report $ PowerpointTemplateWarning $ T.pack ("tableWidth: " ++ show tableWidth)
+  P.report $ PowerpointTemplateWarning $ T.pack ("colWidthsold: " ++ show widths)
+  let colWidths = if any (== 0.0) widths
+                  then if null hdrCells
+                      then case rows of
                          r : _ | not (null r) -> replicate (length r) $
                                                  tableWidth `div` toInteger (length r)
                          -- satisfy the compiler. This is the same as
                          -- saying that rows is empty, but the compiler
                          -- won't understand that `[]` exhausts the
                          -- alternatives.
                          _ -> []
```

Comment:
Maybe more elegant to do
```
  r@(_:_) : _ ->
  [] ->
```
I think the compiler would be happy with that.

Reply:
Almost. I also needed to match empty list in list.
Sorry about the additional checkin; I had a hard time understanding when the widths actually were set so I did some debug printouts which are now removed.